### PR TITLE
Tc05_System shows message when user enters wrong password several times

### PR DESCRIPTION
**Pull request by taile47711**

### CODE REVIEW

File: src/test/java/tests/LoginTest.java [hunk 1]
```diff
@@ -77,4 +77,23 @@ public void tc04_UserIsRedirectedToBookPageAfterLoggingIn() {
...
         Assert.assertEquals(actual, "Book ticket", "Navigate incorrectly to Book ticket page");
     }
+
+    @Test(description = "System shows message when user enters wrong password several times")
+    public void tc05_SystemShowsMessageWhenUserEntersWrongPasswordSeveralTimes() {
+        LogHelper.info("Click login tab");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Enter valid data into Username textbox except Password textbox and click login button. Repeat 3 times");
+        for (int i = 0; i < 4; i++) {
```
Comment (tanmle) [inline src/test/java/tests/LoginTest.java]: wrap this in a method

**Review by tanmle — DISMISSED** [review]

File: src/test/java/tests/LoginTest.java [hunk 2]
```diff
@@ -77,4 +77,23 @@ public void tc04_UserIsRedirectedToBookPageAfterLoggingIn() {
...
+            loginPage.login(Constant.USERNAME, "");
+        }
+
+        LogHelper.info("Get error message");
+        String actual = loginPage.getErrorMessage();
+        String expected = "You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.";
+
+        LogHelper.info("Verify that system shows message when user enters wrong password several times");
+
```
Comment (anhlvt) [inline src/test/java/tests/LoginTest.java]: redundant

Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: Tan and Quan said should be separated like that.

File: src/test/java/tests/LoginTest.java (hunk 1 above)
Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: fixed

File: src/test/java/tests/LoginTest.java (hunk 2 above)
Comment (anhlvt) [reply src/test/java/tests/LoginTest.java]: if so, please make it consistent for all

Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: I will do it

Comment (MinhQuan119) [reply src/test/java/tests/LoginTest.java]: I didn't say that =))))

File: src/test/java/tests/LoginTest.java (hunk 1 above)
Comment (tanmle) [reply src/test/java/tests/LoginTest.java]: what did you fix @taile47711 ?

Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: Sorry about that, I pushed wrong git. Please check again

File: src/main/java/page_object/LoginPage.java [hunk 3]
```diff
@@ -49,4 +49,10 @@ public String getWelcomeMessage() {
     public String getErrorMessage() {
         return this.getLblErrorMessage().getText();
     }
+
+    public void loginMultipleTime(int n, String username, String pasword) {
```
Comment (tanmle) [inline src/main/java/page_object/LoginPage.java]: do not name a var like this, what is n?

File: src/test/java/tests/LoginTest.java (hunk 2 above)
Comment (tanmle) [reply src/test/java/tests/LoginTest.java]: I didn't either

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/test/java/tests/LoginTest.java (hunk 2 above)
Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: maybe I hear wrong. I will fix it

File: src/main/java/page_object/LoginPage.java (hunk 3 above)
Comment (taile47711) [reply src/main/java/page_object/LoginPage.java]: I fixed it. Please check again

File: src/test/java/tests/LoginTest.java (hunk 2 above)
Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: I fixed it. Please check again

File: src/main/java/page_object/LoginPage.java [hunk 4]
```diff
@@ -49,4 +49,10 @@ public String getWelcomeMessage() {
     public String getErrorMessage() {
         return this.getLblErrorMessage().getText();
     }
+
+    public void loginMultipleTime(int loop, String username, String password) {
```
Comment (MinhQuan119) [inline src/main/java/page_object/LoginPage.java]: It's MultipleTimes

Comment (taile47711) [reply src/main/java/page_object/LoginPage.java]: I fixed it. Please check again

**Review by MinhQuan119 — APPROVED** [review]

File: src/main/java/page_object/LoginPage.java (hunk 3 above)
Comment (tanmle) [reply src/main/java/page_object/LoginPage.java]: loop is not a good one.

Comment (taile47711) [reply src/main/java/page_object/LoginPage.java]: Please check again

**Review by tanmle — APPROVED** [review]

**Pull request merged**